README.md

```diff
@@ -18,22 +18,29 @@
 
 ### Sneak peek:
 ```rust,noplaypen
-fn counter() -> impl hobo::Element {
-    let counter = Mutable::new(0);
+pub use hobo::{
+	prelude::*,
+	create as e,
+	signals::signal::{Mutable, SignalExt}
+};
 
-    e::div()
-        .class((
-            css::display!(flex),
-            css::width!(400 px),
-        ))
-        .child(e::div()
-            .text_signal(counter.signal().map(|value| {
-                format!("Counter value is: {}", value);
-            }))
-        )
-        .child(e::button()
-            .text("increment")
-            .on_click(move |_| *counter.lock_mut() += 1)
-        )
+fn counter() -> impl hobo::AsElement {
+	let counter = Mutable::new(0);
+
+	e::div()
+		.class((
+			css::display!(flex),
+			css::flex_direction!(column),
+			css::width!(400 px),
+		))
+		.child(e::div()
+			.text_signal(counter.signal().map(|value| {
+				format!("Counter value is: {}", value)
+			}))
+		)
+		.child(e::button()
+			.text("increment")
+			.on_click(move |_| *counter.lock_mut() += 1)
+		)
```

Comment:
tabs show up as 8-width in mdbook and there's no way to change that soooo all code examples should use 4 spaces for idents